composer.json

```
@@ -4,7 +4,7 @@
     "description": "Flow Form Framework integration into Neos CMS",
     "license": "GPL-3.0+",
     "require": {
-        "neos/neos": "^5.0 || ^7.0 || ^8.0",
+        "neos/neos": "^5.0 || ^7.0 || ^8.0 || ^9.0",
```

This version will only be compatible with 9.x
```suggestion
        "neos/neos": "^9.0",
```

---

You're absolutely right!